src/components/collapsible/Collapsible.js

```diff
@@ -63,10 +75,20 @@ class Collapsible extends React.Component<Props, State> {
             }),
             () => {
                 const { isOpen } = this.state;
-                if (isOpen && onOpen) {
-                    onOpen(this);
-                } else if (!isOpen && onClose) {
-                    onClose(this);
+                if (isOpen) {
```

Comment:
Few things can be cleaned up a bit.

1. Since this function is doing more now, can extract it out at the class level instead of being inlined. Will make it easier to test.
2. onOpen and onClose can be defaulted to lodash/noop, which would reduce the need to test 2 branch statements.
3. this.ref.current is repeated, can be saved as a const/variable.